backend/src/validation/lectureIdParamValidator.ts

```diff
@@ -0,0 +1,5 @@
+import { param } from 'express-validator';
+
+export const lectureIdParamValidator = [
+  param('id').isInt(),
+];
```

Comment:
There are a lot of validators that validate for ID, they all do the same thing:
lectureIdParamValidator, courseIdParamValidator, etc

make a generic idParamValidator that you use for everything